modules/openapi-generator/src/main/resources/rust/request.rs

```diff
@@ -113,7 +113,7 @@ impl Request {
         let mut path = self.path;
         for (k, v) in self.path_params {
             // replace {id} with the value of the id path param
-            {{=<% %>=}}path = path.replace(&format!("{{{}}}", k), &v);<%={{ }}=%>
```

Comment:
This was added in the template management refactor because the file was being processed by Mustache engine (note file is `request.rs` not `request.mustache` and I missed this during the refactor)